src/components/Individual_Article/List_of_Comments/List_of_Comments.jsx

```diff
@@ -0,0 +1,37 @@
+import { useState, useEffect } from "react"
+import { getArticleComments } from "../../../../api"
+
+const CommentsList = ({articleId}) => {
+    const [articleComments, setArticleComments] = useState([])
+    const [isLoading, setLoading] = useState(true)
+    
+    if(articleId){
+        useEffect (() => {
+            getArticleComments(articleId).then((fetchedComments) => {
+                setArticleComments(fetchedComments)
+                setLoading(false)
+            })
+        }, [])
+    }
+    if (isLoading) return <p>Loading...</p>
+
+    return (
+        <section id="commentsSection">
+            <h3 id='commentsHeading'>Comments</h3>
+            <ul id="commentsList">
+                {articleComments.map((comment) => {
```

Comment:
Great use of a `map`, you could turn each comment into it's own component, to separate out the code a bit more and keep your code tidy 🧹